demisto_sdk/__main__.py

```diff
@@ -191,16 +191,16 @@ def unify(**kwargs):
 @click.option(
     '--prev-ver', help='Previous branch or SHA1 commit to run checks against.')
 @click.option(
-    '--post-commit', is_flag=True, help='Whether the validation is done after you committed your files, '
-                                        'this will help the command to determine which files it should check in its '
-                                        'run. Before you commit the files it should not be used. Mostly for build '
-                                        'validations.')
+    '--post-commit', is_flag=True, help='Whether the validation should run only on committed branch changed files.'
+                                        'This applies only when -g flag is supplied')
 @click.option(
     '--no-backward-comp', is_flag=True, show_default=True,
     help='Whether to check backward compatibility or not.')
 @click.option(
     '-g', '--use-git', is_flag=True, show_default=True,
-    default=False, help='Validate changes using git - this will check your branch changes and will run only on them.')
+    default=False, help='Validate changes using git - this will check your branch changes and will run only on them.'
```

Comment:
against origin/master I presume (could be the branch from which it was checked out, what do I know - need to be as clear as possible)? If so, it should say so.